src/views/Leaderboard.vue

```diff
@@ -0,0 +1,187 @@
+<template>
+  <div class="container">
+    <h1>Leaderboard</h1>
+    <div class="board">
+      <!-- heading to select view -->
+      <div class="heading">
+        <button class="active" @click="openPlayers" v-if="showPlayers">Players</button>
+        <button class="inactive" @click="openPlayers" v-if="!showPlayers">Players</button>
+        <div class=divider></div>
+        <button class="active" @click="openProgress" v-if="!showPlayers">My progress</button>
+        <button class="inactive" @click="openProgress" v-if="showPlayers">My progress</button>
+      </div>
+      <!-- Players -->
+      <div class="rows" v-if="showPlayers">
+        <!-- replace with v-for once data is loaded in -->
+        <div class="row">
+          <p class="rank">1</p>
+          <p>Team 1</p>
+          <p class="points">25</p>
+        </div>
+        <div class="row">
+          <p class="rank">2</p>
+          <p>Team 2</p>
+          <p class="points">22</p>
+        </div>
+        <div class="row">
+          <p class="rank">3</p>
+          <p>Team 3</p>
+          <p class="points">20</p>
+        </div>
+        <div class="row">
+          <p class="rank">4</p>
+          <p>Team 4</p>
+          <p class="points">14</p>
+        </div>
+        <div class="row">
+          <p class="rank">5</p>
+          <p>Team 5</p>
+          <p class="points">10</p>
+        </div>
+      </div>
+      <!-- My progress -->
+      <div class="progress" v-if="!showPlayers">
+        <h2>Team 3</h2>
+        <img src="../assets/tree 2.png" alt="Icon">
```

Comment:
Let's avoid adding images into the assets folder. I'll write a ticket to make an S3 instance for LLB (and SFTT).

Reply:
Removed from assets folder